src/atomate2/forcefields/schemas.py

```diff
@@ -0,0 +1,167 @@
+"""Job to prerelax a structure using an MD Potential."""
+
+import logging
+from typing import List
+
+from pydantic import BaseModel, Field
+
+# from jobflow import Maker, job
+from pymatgen.core.structure import Structure
+
+
+class InputDoc(BaseModel):
+    """The inputs used to run this job."""
+
+    structure: Structure = Field(None, description="The inputted structure.")
+    relax_cell: bool = Field(
+        None,
+        description="Whether cell lattice was allowed to change during relaxation.",
+    )
+    relax_kwargs: dict = Field(
+        None,
+        description="Keyword arguments that passed to :obj:`StructOptimizer.relax`.",
+    )
+    optimizer_kwargs: dict = Field(
+        None,
+        description="Keyword arguments that passed to :obj:`StructOptimizer`.",
+    )
+
+
+class OutputDoc(BaseModel):
+    """The outputs of this job."""
+
+    structure: Structure = Field(None, description="The final, relaxed structure.")
+
+    energy: float = Field(None, description="Total energy in units of eV.")
+
+    energy_per_atom: float = Field(
+        None,
+        description="Energy per atom of the final structure in units of eV/atom.",
+    )
+
+    forces: List[List[float]] = Field(
+        None,
+        description="The force on each atom in units of eV/A for the final structure.",
+    )
+
+    # NOTE: units for stresses were converted to kbar
+    # (* -10 from standard output) to comply with MP convention
+    stress: List[float] = Field(
+        None, description="The stress on the cell in units of kbar (in Voigt notation)."
+    )
+
+    trajectory: dict = Field(
+        None, description="Step-by-step trajectory of the structural relaxation."
+    )
```

Comment:
Can we try and emulate the `ionic_steps` document in the VASP Task document. See https://github.com/materialsproject/emmet/blob/2289affb31d9b18aa82e49484ca772a2ce80e934/emmet-core/emmet/core/vasp/calculation.py#LL305C5-L305C114

and 

https://github.com/materialsproject/emmet/blob/2289affb31d9b18aa82e49484ca772a2ce80e934/emmet-core/emmet/core/vasp/calculation.py#L262


So basically, it is a list of dicts rather than multiple dicts of lists. Rather than the funky `e_fr_energy` etc in the IonicStep linked, you can just have `energy`. You can also remove `electronic_steps` and add `magmoms`.

This will involve reorganising the output of CHGNet somewhat but will bring the outputs into line with VASP, CP2K, etc.

Note that we should also store the structure at each step. You can convert an ase atoms object using:
```python
from pymatgen.io.ase import AseAtomsAdaptor
structure = AseAtomsAdaptor.get_structure(atoms)
```

Reply:
so you'd want one of the output fields to be `ionic_steps`, which is a list. That list will contain as many dicts as are specifies, which would include things like `energy`, `structure`, etc.? Sure I can do that.

Separately--considering that the linked VASP Task Document doesn't even let users specify which parts of the trajectory to keep, I don't think we should either. I'll just replace the tuple/list `keep_info` with a bool `keep_trajectory`. Let me know if you have any objections.

Reply:
Actually, it appears that users don't even have the option to _not_ store the trajectory for MD jobs. Seems pointless to exclude it for FF relaxations in that case. I'll just remove the tag entirely.

Reply:
RE: converting the `Atoms` objects to structures. It appears that CHGNet only outputs a single `Atoms` object, rather than one for each step... very odd. Any advice?

Reply:
I think having the keep_info tag is actually useful and just because its not there in VASP doesn't mean we can't add it here. So I would keep that in and forget about `keep_trajectory` for now. However, one we add in MD through force fields, we will want to add trajectory support (a latter PR).

You can generate a new structure from the positions and cells along the lines of:
```python
species = atoms.get_chemical_symbols()
for pos, cell in zip(atom_positions, cells):
    structure = Structure(cell, species, positions, coords_are_cartesian=False)
```

Double check whether the `atom_positions` are in cartesian coordinates and update `coords_are_cartesian` accordingly.

Reply:
Ah, I didn't know about the `coords_are_cartesian` input, and missed it when looking through the documentation. That fixed it.